interop-rx2/src/main/kotlin/com/bumble/appyx/interop/rx2/plugin/DisposeOnDestroyPlugin.kt

```diff
@@ -0,0 +1,17 @@
+package com.bumble.appyx.interop.rx2.plugin
+
+import com.bumble.appyx.core.plugin.Destroyable
+import com.bumble.appyx.core.plugin.Plugin
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.disposables.Disposable
+
+private class DisposeOnDestroyPlugin(disposables: List<Disposable>) : Destroyable {
```

Comment:
we don't use Plugin suffix for other plugin, let's make it consistent?

Reply:
We probably should. Though as this is private, I'm happy to do this for :)